src/ClientApp/src/GemeindenMap.js

```diff
@@ -0,0 +1,128 @@
+export const GemeindenMap = {
```

Comment:
PP: In einem nächsten Schritt sollten wir die `GemeindeMap` dynamisch zusammenbauen (z.B. über das [swisstopo REST API](https://api3.geo.admin.ch/rest/services/api/MapServer/find?layer=ch.swisstopo-vd.geometa-gemeinde&searchText=Solothurn&searchField=kanton&returnGeometry=false))